[Management] Update management landing page copy

alisonelizabeth opened the pull request:
Fixes https://github.com/elastic/kibana/issues/163997

Note: The "Endpoints" card copy mentioned in the issue will be added in a separate PR. Also, I'm getting clarification on the "Logs" card for Observability; if we need to add it, it can be done via a separate PR.

<img width="1033" alt="Screenshot 2023-09-19 at 9 46 44 AM" src="https://github.com/elastic/kibana/assets/5226211/f2c44c4a-c068-4986-9bfd-d27f7319b192">

elasticmachine commented:
Pinging @elastic/platform-deployment-management (Team:Deployment Management)

sabarasaba reviewed (approved):
🚀

lcawl commented:
IMO the description for the "Connectors" is still possibly confusing for folks who are seeking the search-related connectors. What about something like "Configure connections to third party systems for use in cases and rules."

@shanisagiv1

alisonelizabeth commented:
> IMO the description for the "Connectors" is still possibly confusing for folks who are seeking the search-related connectors. What about something like "Configure connections to third party systems for use in cases and rules."

I'm happy to update the copy to this. @gchaps wdyt?

gchaps commented:
+1 to 

> Configure connections to third party systems for use in cases and rules.

gchaps reviewed (commented):
In looking over this, I felt that the intro was a too short and the descriptions read better when they are 2 lines.  Here are my suggestions.  I am interested in what others think.

**Intro**

Manage data and indices, oversee rules and connectors, organize saved objects and files, and create API keys in a central location.

**Cards**

Configure and maintain your Elasticsearch indices for data storage and retrieval.

Pivot your data or copy the latest documents into an entity-centric index.

Remove fields, extract values, and perform transformations on your data.

Identify, analyze, and process your data using advanced data analysis.

Manage and view the Logstash event processing pipeline from inputs to outputs.

Detect conditions in apps across Kibana and trigger actions.

Configure connections to third party systems for use in cases and rules.

Manage your saved dashboards, maps, data views, and Canvas workloads.

Monitor the generation of PDF, PNG and CSV reports.

Organize, search, and filter your saved objects by specific criteria.

@lcawl @sixstringcode

shanisagiv1 commented:
++
I think since it's under the "Alert & Insights" it's clear that the connectors refer to the alerts but fine with this update.

But since the ML is not part of this section anymore (as it is in stateful).. I'd remove the word "insights" and keep it just "Alerts" - wdyt?

alisonelizabeth commented:
@gchaps thanks for the review! I believe I addressed all the feedback. Mind taking another look?

<img width="868" alt="Screenshot 2023-09-26 at 9 19 40 PM" src="https://github.com/elastic/kibana/assets/5226211/606f869c-57c4-454d-924e-d35cf244506e">

gchaps reviewed (approved):
Canvas workloads > Canvas workpads.  Sorry about that.

Otherwise, text LGTM.

@lcawl, @sixstringcode What do you think of the new intro and shorter text in the cards?

lcawl commented:
IMO we should try to remove the "Kibana" reference from the Rules intro.  How about something like "Define when to generate alerts and notifications".

Likewise I don't think the "Reporting" app enables you to do any monitoring. How about: "Manage generated PDF, PNG and CSV reports".

sixstringcode commented:
> ++ I think since it's under the "Alert & Insights" it's clear that the connectors refer to the alerts but fine with this update.
> 
> But since the ML is not part of this section anymore (as it is in stateful).. I'd remove the word "insights" and keep it just "Alerts" - wdyt?

I think we should keep it, I'll be adding a few things for Security. Entity Risk only make sense if it still says insights

sixstringcode commented:
> IMO we should try to remove the "Kibana" reference from the Rules intro. How about something like "Define when to generate alerts and notifications".
> 
> Likewise I don't think the "Reporting" app enables you to do any monitoring. How about: "Manage generated PDF, PNG and CSV reports".

+1 I have been actively removing references to Kibana as well. It doesn't have any context in the new serverless IA's.

sixstringcode commented:
@gchaps the new shorter ones look much better. It's VERY minor, but I could see modifying Data Views a little: "Create and manage data you explore from Elasticsearch" or something like that. Object feels like a technical detail we don't need.

We should also capture the ones we haven't added yet here. Advanced Settings we have already discussed, but it looks like we will add Maps, Entity Risk and Visualize Library for Security Projects. We'll need copy for those also. https://github.com/elastic/kibana/issues/167453

gchaps commented:
@sixstringcode How about one of these:

> Create and manage the data you retrieved for exploration.
> Create and manage the Elasticsearch data you retrieved for exploration.

alisonelizabeth commented:
I've addressed the latest feedback in https://github.com/elastic/kibana/pull/166732/commits/271ca6ec4989266e8630f7fe34abb17abce30c34.

Please advise on the final copy for Data Views.

> We should also capture the ones we haven't added yet here. Advanced Settings we have already discussed, but it looks like we will add Maps, Entity Risk and Visualize Library for Security Projects. We'll need copy for those also. https://github.com/elastic/kibana/issues/167453

I'm 100% on finalizing copy for these new cards, however, I think the actual code change is out of scope for this PR.

gchaps commented:
@szabosteve gave me a minor edit to the ML card description so that the word "data" is not repeated:

> Identify, analyze, and process your data using advanced analysis techniques.

sixstringcode commented:
> I've addressed the latest feedback in [271ca6e](https://github.com/elastic/kibana/commit/271ca6ec4989266e8630f7fe34abb17abce30c34).
> 
> @sixstringcode please advise on the final copy for Data Views.
> 
> > We should also capture the ones we haven't added yet here. Advanced Settings we have already discussed, but it looks like we will add Maps, Entity Risk and Visualize Library for Security Projects. We'll need copy for those also. #167453
> 
> I'm 100% on finalizing copy for these new cards, however, I think the actual code change is out of scope for this PR.

Fair, I'll take care of them on https://github.com/elastic/kibana/issues/167453

sixstringcode commented:
> @sixstringcode How about one of these:
> 
> > Create and manage the data you retrieved for exploration.
> > Create and manage the Elasticsearch data you retrieved for exploration.

I know it's being picky, but the data comes from Elasticsearch. It's really is a filter of the same data. Retrieved makes it sound like it's been moved or copied. Maybe selected or defined?

alisonelizabeth commented:
Final copy feedback applied via https://github.com/elastic/kibana/pull/166732/commits/9c01f3372fedc9600287fddb509dbb0f60c907f0
